source/administration/encryption.rst

```diff
@@ -0,0 +1,34 @@
+Encryption Options
+=======================
+
+Mattermost provides encryption-in-transit and encryption-at-rest for both database and file storage. This page guides you through setting up appropriate encryption security.
+
+Encryption is not required for GDPR, although it can be used as an additional safeguard against data breach.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+Encryption-in-transit
+-----------------------
+
+Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet. You may either set up TLS on the Mattermost Server or install a proxy such as NGINX and set up TLS on the proxy. Refer to our `configuration guide for more details <https://docs.mattermost.com/install/config-tls-mattermost.html>`_.
+
+Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel <https://docs.mattermost.com/administration/config-settings.html#id11>`_ in Enterprise Edition E10 and E20.
+
+Encryption-at-rest
+-----------------------
+
+Database
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable end user search and compliance reporting of message histories, Mattermost does not offer native encryption-at-rest within the database.
```

Comment:
Seems maybe unnecessary to say we don't support native encryption at rest. Why not go right to the solution? Start with  "Encryption-at-rest is available for messages via hardware and software disk encryption solutions applied to the Mattermost database..."